### ISSUE TITLE
Passing multiple files to the plugin for version bumping

### ISSUE THREAD

**Issue by ohmycaptainnemo** [monim67/poetry-bumpversion#1]
Hi @monim67 

Love your work.

Correct me if I am mistaken, but currently, if one would like to bump the version in say two separate files (i.e. file_1.py and file_2.py), one needs to add two code blocks like this:

```
[tool.poetry_bumpversion.file."file_1.py"]
search = '__version__ = "{current_version}"'
replace = '__version__ = "{new_version}"'

[tool.poetry_bumpversion.file."file_2.py"]
search = '__version__ = "{current_version}"'
replace = '__version__ = "{new_version}"'
```

I was hoping we could avoid this repetition and instead of doing what I showed above, we do something like this:

```
[tool.poetry_bumpversion]
files = ["file_1.py", "file_2.py"]
search = '__version__ = "{current_version}"'
replace = '__version__ = "{new_version}"'
```

What are your thoughts?

**Comment by monim67** [comment]
I don't like repeating either. Taking `files` as array makes a lot sense here. One problem is, the search/replace keywords has to be same for all the files, which might not be the case always.

I'd suggest an extension of your proposition

```toml
[[tool.poetry_bumpversion.replacements]]
files = ["file_1.py", "file_2.py"]
search = '__version__ = "{current_version}"'
replace = '__version__ = "{new_version}"'

[[tool.poetry_bumpversion.replacements]]
files = ["README.md"]
search = 'version: "{current_version}"'
replace = 'version: "{new_version}"'
```

This way we can also keep the backward compatibility of `tool.poetry_bumpversion.file` syntax.

**Comment by ohmycaptainnemo** [comment]
> I don't like repeating either. Taking `files` as array makes a lot sense here. One problem is, the search/replace keywords has to be same for all the files, which might not be the case always.
> 
> I'd suggest an extension of your proposition
> 
> ```toml
> [[tool.poetry_bumpversion.replacements]]
> files = ["file_1.py", "file_2.py"]
> search = '__version__ = "{current_version}"'
> replace = '__version__ = "{new_version}"'
> 
> [[tool.poetry_bumpversion.replacements]]
> files = ["README.md"]
> search = 'version: "{current_version}"'
> replace = 'version: "{new_version}"'
> ```
> 
> This way we can also keep the backward compatibility of `tool.poetry_bumpversion.file` syntax.

Sounds good.

**Comment by monim67** [comment]
This feature is now available in version [0.2.0](https://github.com/monim67/poetry-bumpversion/releases/tag/0.2.0)

**Comment by ohmycaptainnemo** [comment]
> This feature is now available in version [0.2.0](https://github.com/monim67/poetry-bumpversion/releases/tag/0.2.0)

Thank you for the legendary effort.

**Comment by monim67** [comment]
This version uses the same method as previous version, it just adds a new feature to it. So previous features should work as it was. I can also install it with no issues.
